Makefile

```diff
@@ -21,7 +21,7 @@ MAKEFLAGS += --no-print-directory
 LD = ld
 CC = gcc
 CFLAGS = -Wall -I../include
-LFLAGS = -lpthread
+LFLAGS = -lpthread -pthread
```

Comment:
`LFLAGS = -pthread` is correct format. Could you update your pull request.
More details about `-pthread` option can be [here](http://stackoverflow.com/a/1665110/1009249).